pages/cards.html

```diff
@@ -40,5 +39,6 @@
         </div>
       </main>
     </div>
+    <script src="../scripts/cards.js" defer></script>
```

Comment:
here the script tag should always be at the bottom so the page can load, rather than the page waiting for the scrips tag to load.